test/test_basic.hpp

```diff
@@ -33,6 +33,77 @@ LTESTexe(a_plus_b, {
         TEST_EQ(tested(A[n],B[n]), A[n]+B[n]);
 });
 
+IReg __pow__(const IReg& _x, int p, loops::Context& CTX)
+{
+    USE_CONTEXT_(CTX);
```

Comment:
1. Can you move the loop inside the generated function? (so that it processes array, not a single number?)
2. Is it possible to make a single function with "int p" parameter that would generate pow function for the given p? Now we have STARTFUNC duplicated 3 times